release-1.8/release_notes_draft.md

```diff
@@ -271,18 +278,35 @@ kind.
 
 #### Storage
 
-* Capacity Isolation/Resource Management for Local Ephemeral Storage
-* Block Volumes Support
-* Enable containerization of mount dependencies
-* Support Attach/Detach for RWO volumes such as iSCSI, Fibre Channel and RBD
-* Volume Plugin Metrics
-* Snapshots
-* Resizing Volume Support
-* Exposing StorageClass Params To End Users (aka Provisioning configuration in PVC)
-* Mount Options to GA
-* Allow configuration of reclaim policy in StorageClass
-* Expose Storage Usage Metrics
-* PV spec refactoring for plugins that reference namespaced resources: Azure File, CephFS, iSCSI, Glusterfs
+* [stable] Mount Options
+  * The ability to speciy mount options has been promoted to stable.
+  * A new `MountOptions` field in the `PersistentVolume` spec can be used to specify mount options (instead of an annotation).
+* [stable] Support Attach/Detach for RWO volumes iSCSI and Fibre Channel.
+* [stable] Expose Storage Usage Metrics
+  * Expose how much available capacity a given PV has through the Kubernetes metrics API.
+* [stable] Volume Plugin Metrics
+  * Expose success and latency metrics for all Kubernetes mount/unmount/attach/detach/provision/delete calls through the Kubernetes metrics API.
+* [stable] Modify PV spec for Azure File, CephFS, iSCSI, Glusterfs to allow referencing namespaced resources.
+* [beta] Reclaim policy in StorageClass
+  * Allow configuration of reclaim policy in StorageClass, instead of always defaulting to `delete` for dynamically provisioned volumes.
+* [alpha] Resizing of Volumes
+  * Enable increasing the size of Volumes through the Kubernetes API.
+  * For alpha, this feature only increases the size of the underlying volume and does not do filesystem resizing.
+  * For alpha, this feature is only implmented for Gluster volumes.
+* [alpha] Capacity Isolation/Resource Management for Local Ephemeral Storage:
+  * Introduce ability to set container requests/limits and node Alloctable reservations for new `ephemeral-storage` resource.
+  * The `ephemeral-storage` resource includes all the disk space space a container may use (via container overlay or scratch).
+* [alpha] Mount namespace propagation
+  * Introduce new `VolumeMount.Propagation` field for `VolumeMount` in pod containers.
+  * This field may be set to `Bidirectional` to enable a particular mount for a container to be propagated from the container to the host or other containers.
+* [alpha] Improve Flexvolume Deployment
+  * Make it easier to deploy Flex volume drivers:
+    * Automatically discover and initialize new driver files instead of requiring kubelet/controller-manager restart.
+    * Provide a sample DaemonSet that can be used to deploy Flexvolume drivers.
+* [prototype] Volume Snapshots
+  * Enable triggering a volume snapshoting through the Kubernetes API.
```

Comment:
maybe " Enable triggering the creation of a volume snapshot through .." ?

Reply:
Much nicer. Done.